### PR TITLE
Bug/702 br tags in sequence diagrams

### LINKED ISSUE #702: Sequencediagram text with <br /> BR tag with space fails
This works 

[working example](https://mermaidjs.github.io/mermaid-live-editor/#/view/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtXG4gICAgcGFydGljaXBhbnQgSm9lXG4gICAgcGFydGljaXBhbnQgQWxpY2VcbiAgICBKb2UtPj5BbGljZTogRm9vXG4gICAgTm90ZSBvdmVyIEFsaWNlOiBCYXI8YnIvPkJhelxuICAgIE5vdGUgb3ZlciBKb2UsQWxpY2U6IFRoYXQgbm90ZSBhYm92ZSBzaG91bGQgaGF2ZSBoYWQgYSBjYXJyaWFnZSByZXR1cm4gYmV0d2VlbiBCYXIgYW5kIEJheiIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In19)

```
sequenceDiagram
    participant Joe
    participant Alice
    Joe->>Alice: Foo
    Note over Alice: Bar<br/>Baz
    Note over Joe,Alice: That note above should have had a carriage return between Bar and Baz
```
but this, with a space between the `r` and `/` fails by rendering the `<br />` as text not as a line break. One tricky thing is that if you look at it in the interactive view, and then type something on the line with the `<br />` it displays correctly, but the view link continues to display incorrectly.

[non working example](https://mermaidjs.github.io/mermaid-live-editor/#/view/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtXG4gICAgcGFydGljaXBhbnQgSm9lXG4gICAgcGFydGljaXBhbnQgQWxpY2VcbiAgICBKb2UtPj5BbGljZTogRm9vXG4gICAgTm90ZSBvdmVyIEFsaWNlOiBCYXI8YnIgLz5CYXpcbiAgICBOb3RlIG92ZXIgSm9lLEFsaWNlOiBUaGF0IG5vdGUgYWJvdmUgc2hvdWxkIGhhdmUgaGFkIGEgY2FycmlhZ2UgcmV0dXJuIGJldHdlZW4gQmFyIGFuZCBCYXoiLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9fQ)

```
sequenceDiagram
    participant Joe
    participant Alice
    Joe->>Alice: Foo
    Note over Alice: Bar<br />Baz
    Note over Joe,Alice: That note above should have had a carriage return between Bar and Baz
```

This may relate to some challenges people had in #384

### PR DESCRIPTION
**Pull request by GDFaber**
## Summary
Improved regex for multiline texts to find ```<br />``` as well as ```<br/>``` and ```<br>```.

## Design Descisions
Improved regex for multiline texts in sequence diagram renderer.

Resolves #702

### CODE REVIEW

**Comment by knsv** [comment]
Nice, leaving mermaid a little better!

File: src/diagrams/sequence/sequenceRenderer.js [hunk 1]
```diff
@@ -168,7 +168,7 @@ export const bounds = {
 
 const _drawLongText = (text, x, y, g, width) => {
   let textHeight = 0;
-  const lines = text.split(/<br\/?>/gi);
+  const lines = text.split(/<br ?\/?>/gi);
```
Comment (gene1wood) [inline src/diagrams/sequence/sequenceRenderer.js]: It think what's needed here, to follow [the spec](http://www.w3.org/TR/REC-xml/#sec-starttags) isn't `<br ?\/?>` but instead `<br\s*\/?>` as HTML allows an arbitrary amount of whitespace (as opposed to 0 or 1 characters of whitespace) between the end of an HTML tag and that whitespace can be space tab carriage return or line feed.

Comment (gene1wood) [reply src/diagrams/sequence/sequenceRenderer.js]: I'll see if I can PR that minor tweak today

Comment (GDFaber) [reply src/diagrams/sequence/sequenceRenderer.js]: There's more places than this in mermaid where regex for line breaks is used. These differ in their abilities to find certain line break notations. Maybe this should be a new issue to get this done for all mermaid diagram types.

Comment (gene1wood) [reply src/diagrams/sequence/sequenceRenderer.js]: Good point, I'll take a look and open an issue (or a PR if it's easily solvable)

**Pull request merged**